src/app/cube/details/details.component.html

```diff
@@ -41,7 +41,7 @@
       <div class="inner"*ngIf="reviewer">
         <div class="revisions">
```

Comment:
Should this element be the one to have the `*ngIf="hasRevisions"` since the View Revisions div and the clark-version-switch component should only show if `hasRevisions` is true?

Reply:
Yes, I changed that with change logic commit. I didn't realize I didn't copy and paste in the right div.

Reply:
I must have opened the review right before you pushed. I see it now 👍